### PR TITLE
Add instance bulk deletion

### PR DESCRIPTION
**Pull request by edlerd**
## Done

- add bulk delete of instances

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described in the [Readme](https://github.com/canonical/lxd-ui#setting-up-for-development).
2. Perform the following QA steps:
    - bulk delete instances

### CODE REVIEW

**Comment by al-bakalova** [comment]
@edlerd This looks good. I have two small comments for you:

- The clear selection button seems not to be aligned. Yet, it is only two pixels so we can also ignore it.
Screenshot 2023-05-11 at 13 37 38

- I would bring the Delete button closer to the actions, I think half of the current space should be enough

- Resizing the screen, I get bigger buttons with the label going to the next line. Was it always like that?
Screenshot 2023-05-11 at 13 39 16

I would suggest to keep only the icons or just the labels.  When the buttons take two lines the layout seems a little broken.
Can we please fix it? I am happy to create a new task for this!

**Comment by al-bakalova** [comment]
@edlerd Also, when we tell the user some instances will not be affected, we still tend to disable them while the operation is taking action. Would it be possible, as a future improvement, to disable only the ones where an operation is currently running and enable the rest?

**Comment by al-bakalova** [comment]
@edlerd This looks perfect! 

On a side note - I have tried to test the functionality itself and I keep on getting this error. I am not sure how to understand it, so I wanted to make sure you were aware:
Screenshot 2023-05-12 at 9 29 09

**Comment by edlerd** [comment]
> On a side note - I have tried to test the functionality itself and I keep on getting this error. I am not sure how to understand it, so I wanted to make sure you were aware:

Good that you mentioned it, the timeout for delete was just 10 seconds, which might too low for the bulk deletion case. I increased it to 2 minutes for the bulk delete.

File: src/pages/instances/actions/InstanceBulkDelete.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  onFinish: () => void;
+}
+
+const InstanceBulkDelete: FC<Props> = ({ instances, onStart, onFinish }) => {
+  const notify = useNotify();
+  const queryClient = useQueryClient();
+  const [isLoading, setLoading] = useState(false);
+
+  const deletableInstances = instances.filter(
+    (instance) => instance.status === "Stopped"
```
[the remark this answers is not in the document]
Comment (edlerd) [reply src/pages/instances/actions/InstanceBulkDelete.tsx]: Added it here and to the single delete action.

**Review by lorumic — APPROVED** [review]

**Pull request merged**